docs/migration/5_0-generated-typescript-changes-doc.md

```diff
@@ -276,6 +276,38 @@ constructor(
 ```
 
 
+### Constructor changed.
```

Comment:
not related to this line, but there's a duplicate entry for `ConfiguratorAttributeDropDownComponent`

Reply:
nvm my mistake :)